client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -958,6 +959,37 @@ public void testClearScroll() throws IOException {
         assertEquals(REQUEST_BODY_CONTENT_TYPE.mediaTypeWithoutParameters(), request.getEntity().getContentType().getValue());
     }
 
+    public void testExistsAlias() {
+        GetAliasesRequest getAliasesRequest = new GetAliasesRequest();
+        String[] indices = randomIndicesNames(0, 5);
+        getAliasesRequest.indices(indices);
+        String[] aliases = randomIndicesNames(0, 5);
```

Comment:
Should this be `randomIndicesNames(1, 5)` instead?

Reply:
no the idea was to also test the case where the aliases array is empty, that is supported and should be interpreted as "all aliases"

Reply:
I though this for `GET` rather than for `HEAD`? In the specs the alias is set to `required`. Am I missing something?

Reply:
the SPEC may be outdated. the code and behaviour is the same for HEAD or GET in the server side, we just don't return a response body when HEAD is used.

Reply:
Oh, I see now.. My point was for the corner case when there is no alias and no index, then only `GET _alias` is supported, isn't it?

Reply:
you are right, thanks for pointing this out.